tests/workflow_tests/test_nph_sms_workflows.py

```diff
@@ -457,3 +457,25 @@ def test_sms_pipeline_n1_function(self, task_mock):
                                      payload=data,
                                      queue='nph')
 
+
+    def test_sms_ingestion_filetype_failure(self):
```

Comment:
im not sure how this is testing a filetype failure. If this is testing that an error is occuring pls add that to the name like in L386. If you can also add comments. I am not sure what you are trying to test for here.